tests/h/search/query_test.py

```diff
@@ -3,188 +3,182 @@
+    def filtered_search(self, search):
+        search.append_filter(query.AuthorityFilter("auth1"))
+        return search
 
 
 class TestAuthFilter(object):
-    def test_logged_out_user_can_not_see_private_annotations(self, search, Annotation):
+    def test_logged_out_user_can_not_see_private_annotations(self, filtered_search, Annotation):
         Annotation()
         Annotation()
 
-        result = search.run({})
+        result = filtered_search.run({})
 
         assert not result.annotation_ids
 
-    def test_logged_out_user_can_see_shared_annotations(self, search, Annotation):
+    def test_logged_out_user_can_see_shared_annotations(self, filtered_search, Annotation):
         shared_ids = [Annotation(shared=True).id,
                       Annotation(shared=True).id]
 
-        result = search.run({})
+        result = filtered_search.run({})
 
         assert set(result.annotation_ids) == set(shared_ids)
 
-    def test_logged_in_user_can_only_see_their_private_annotations(self, search, pyramid_config, Annotation):
+    def test_logged_in_user_can_only_see_their_private_annotations(self,
+            filtered_search, pyramid_config, Annotation):
         userid = "acct:bar@auth2"
         pyramid_config.testing_securitypolicy(userid)
         # Make a private annotation from a different user.
         Annotation(userid="acct:foo@auth2").id
         users_private_ids = [Annotation(userid=userid).id,
                              Annotation(userid=userid).id]
 
-        result = search.run({})
+        result = filtered_search.run({})
 
         assert set(result.annotation_ids) == set(users_private_ids)
 
-    def test_logged_in_user_can_see_shared_annotations(self, search, pyramid_config, Annotation):
+    def test_logged_in_user_can_see_shared_annotations(self,
+            filtered_search, pyramid_config, Annotation):
         userid = "acct:bar@auth2"
         pyramid_config.testing_securitypolicy(userid)
         shared_ids = [Annotation(userid="acct:foo@auth2", shared=True).id,
                       Annotation(userid=userid, shared=True).id]
 
-        result = search.run({})
+        result = filtered_search.run({})
 
         assert set(result.annotation_ids) == set(shared_ids)
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        # We don't need to append AuthFilter to Search because it's one of the filters that
-        # Search appends by default.
-        return search.Search(pyramid_request)
+    def filtered_search(self, search, pyramid_request):
+        search.append_filter(query.AuthFilter(pyramid_request))
+        return search
 
 
 class TestGroupFilter(object):
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        # We don't need to append GroupFilter to Search because it's one of the filters that
-        # Search appends by default.
-        return search.Search(pyramid_request)
+    def filtered_search(self, search):
+        search.append_aggregation(query.GroupFilter())
```

Comment:
Why is this `append_aggregation` and not `append_filter`?

Reply:
Me too...lol I thought I removed all of those but I guess I didn't. I'll fix that.